### PR TITLE
UI: ImageTextColumn

### PR DESCRIPTION
**Pull request by TeamBattino**
image
image

### CODE REVIEW

File: components/Sections/ImageTextColumns/ImageTextColumns.tsx [hunk 1]
```diff
@@ -35,19 +60,20 @@ export async function ImageTextColumnsBuilder({
...
-          <ImageTextColumnsImageBuilder id={imageTextColumns.leftImage} />
-        )
-      }
-      rightImageSlot={
-        imageTextColumns.rightImage && (
-          <ImageTextColumnsImageBuilder id={imageTextColumns.rightImage} />
-        )
-      }
-    />
+    <>
```
Comment (silvio2402) [inline components/Sections/ImageTextColumns/ImageTextColumns.tsx:71]: Remove unnecessary fragment

File: components/Sections/ImageTextColumns/ImageTextColumns.tsx [hunk 2]
```diff
@@ -1,24 +1,49 @@
...
+            className={`rounded-lg overflow-hidden md:basis-[30%] grow-${leftImageInfo.width}`}
+          >
+            <ImageComponent
+              path={leftImageInfo.path}
+              title={leftImageInfo.title || ""}
+            />
+          </div>
+        )}
+        <div className='bg-accent rounded-lg p-4 shrink min-w-0'>
+          {content && <WYSIWYG content={content} style={style}></WYSIWYG>}
```
Comment (silvio2402) [inline components/Sections/ImageTextColumns/ImageTextColumns.tsx:34]: Use a self closing tag for the `WYSIWYG`

File: components/Sections/ImageTextColumns/ImageTextColumns.tsx [hunk 3]
```diff
@@ -1,24 +1,49 @@
...
-      {content && <div dangerouslySetInnerHTML={{ __html: content }} />}
-      {rightImageSlot}
+      <div className='flex flex-col md:flex-row items-stretch gap-4'>
+        {leftImageInfo && (
+          <div
+            className={`rounded-lg overflow-hidden md:basis-[30%] grow-${leftImageInfo.width}`}
+          >
+            <ImageComponent
+              path={leftImageInfo.path}
+              title={leftImageInfo.title || ""}
```
Comment (silvio2402) [inline components/Sections/ImageTextColumns/ImageTextColumns.tsx:29]: Either make the `title` prop optional, or make it required in Directus.

File: components/Sections/ImageTextColumns/ImageTextColumns.tsx [hunk 4]
```diff
@@ -1,24 +1,49 @@
...
+        <div className='bg-accent rounded-lg p-4 shrink min-w-0'>
+          {content && <WYSIWYG content={content} style={style}></WYSIWYG>}
+        </div>
+        {rightImageInfo && (
+          <div
+            className={`rounded-lg overflow-hidden md:basis-[30%] grow-${rightImageInfo.width}`}
+          >
+            <ImageComponent
+              path={rightImageInfo.path}
+              title={rightImageInfo.title || ""}
```
Comment (silvio2402) [inline components/Sections/ImageTextColumns/ImageTextColumns.tsx:42]: Either make the `title` prop optional, or make it required in Directus.

**Review by silvio2402 — CHANGES_REQUESTED** [review]

**Comment by silvio2402** [comment]
@cediackermann add the ImageTextColumns Section to the M2A relation on the Pages collection

File: components/Sections/ImageTextColumns/ImageTextColumns.tsx [hunk 5]
```diff
@@ -1,24 +1,49 @@
...
+            />
+          </div>
+        )}
+      </div>
```
[the remark this answers is not in the document]
Comment (TeamBattino) [reply components/Sections/ImageTextColumns/ImageTextColumns.tsx]: Made some changes, but you're still very much right.

File: lib/fonts.ts [hunk 6]
```diff
@@ -11,5 +11,4 @@ export const poppins = Poppins({
   weight: ["400", "700"],
   variable: "--poppins",
   display: "swap",
-  subsets: ["latin"],
```
[the remark this answers is not in the document]
Comment (TeamBattino) [reply lib/fonts.ts:14]: This seems to be a reocurring issue wehen not using the --turbo option. This is also the reason the CI build is currently failing...

File: backend/directus/snapshot.yml [hunk 7]
```diff
@@ -1,5 +1,5 @@
 version: 1
-directus: 10.9.0
+directus: 10.13.2
```
Comment (silvio2402) [inline backend/directus/snapshot.yml:2]: First update Directus version in docker compose file.

File: backend/directus/snapshot.yml [hunk 8]
```diff
@@ -6097,10 +6117,14 @@ fields:
...
-          - path:
-              _nends_with: /
+          - _or:
+              - path:
+                  _eq: /
+              - _and:
+                  - path:
+                      _nstarts_with: /
+                  - path:
+                      _nends_with: /
```
Comment (silvio2402) [inline backend/directus/snapshot.yml:6127]: Good.

File: backend/directus/snapshot.yml [hunk 9]
```diff
@@ -6111,7 +6135,7 @@ fields:
       max_length: 255
       numeric_precision: null
       numeric_scale: null
-      is_nullable: false
+      is_nullable: true
```
Comment (silvio2402) [inline backend/directus/snapshot.yml:6138]: Why make this nullable?

File: components/Shared/ImageComponent.tsx [hunk 10]
```diff
@@ -1,17 +1,17 @@
...
+  title: string | null;
 }
 
-export function ImageComponent({ path, title }: ImageComponentsProps) {
+export function ImageComponent({ path, title }: ImageComponentProps) {
   return (
     <Image
       src={path}
-      alt={title}
+      alt={title || path}
```
Comment (silvio2402) [inline components/Shared/ImageComponent.tsx:12]: Just make `alt` undefined when there is no title.

**Review by silvio2402 — CHANGES_REQUESTED** [review]

File: components/Shared/ImageComponent.tsx (hunk 10 above)
Comment (TeamBattino) [reply components/Shared/ImageComponent.tsx:12]: impossible!

File: backend/directus/snapshot.yml (hunk 9 above)
Comment (TeamBattino) [reply backend/directus/snapshot.yml:6138]: to set the default homepage

**Review by silvio2402 — APPROVED** [review]
I hope it works

**Pull request merged**